composer.json

```diff
@@ -13,7 +13,7 @@
         "doctrine/collections": "^1.6",
         "doctrine/common": "^3.1",
         "doctrine/dbal": "^2.13 || ^3.0",
-        "doctrine/event-manager": "^1.1",
+        "doctrine/event-manager": "^1.2 || ^2.0",
```

Comment:
The 1.1 to 1.2 bump is needed in order to be able to have code that is compatible with both v1 and v2 at the same time (the `getAllListeners` method call to be more specific).